packages/@blockprotocol/type-system/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@blockprotocol/type-system",
+  "version": "0.0.1",
+  "description": "Definitions of types within the Block Protocol Type System",
+  "homepage": "https://blockprotocol.org",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:blockprotocol/blockprotocol.git"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "HASH",
+    "url": "https://hash.ai"
+  },
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/cjs/index.d.ts",
+      "node": "./dist/node/index.cjs",
+      "import": "./dist/es/index.js",
+      "default": "./dist/cjs/index.cjs"
+    },
+    "./slim": {
+      "types": "./dist/cjs-slim/index-slim.d.ts",
+      "node": "./dist/node/index.cjs",
+      "import": "./dist/es-slim/index-slim.js",
+      "default": "./dist/cjs-slim/index-slim.cjs"
+    },
+    "./type-system.wasm": "./dist/wasm/type-system.wasm",
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/umd/index.js",
+  "module": "./dist/es/index.js",
+  "types": "./dist/umd/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn build:wasm && yarn optimize && yarn build:bundle",
+    "build:bundle": "rollup -c --bundleConfigAsCjs",
+    "build:wasm": "(cd ../../../crates/type-system && cargo make build)",
+    "clean": "rimraf ./dist/",
+    "compressed-size": "yarn build && find dist -iname '*.js' -exec npx terser@latest --compress --mangle --output {} -- {} \\;",
+    "fix:eslint": "eslint --fix .",
+    "lint:eslint": "eslint .",
+    "lint:tsc": "tsc --noEmit",
+    "optimize": "wasm-opt -Os wasm/type-system_bg.wasm -o tmp.wasm && mv tmp.wasm wasm/type-system_bg.wasm",
+    "prepublishOnly": "yarn build:wasm && yarn test && yarn optimize && yarn build:bundle",
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@local/eslint-config": "0.0.0-private",
+    "@local/tsconfig": "0.0.0-private",
+    "@rollup/plugin-typescript": "9.0.2",
+    "@rollup/plugin-wasm": "6.0.1",
+    "@types/jest": "29.2.3",
+    "@types/node": "18.11.10",
+    "eslint": "8.28.0",
+    "jest": "29.3.1",
+    "rollup": "3.5.1",
+    "ts-jest": "29.0.3",
+    "tslib": "2.4.1",
+    "typescript": "4.9.3"
+  },
+  "collaborators": [
+    "HASH"
+  ]
```

Comment:
Not sure we need this section. Might be worth comparing this file with `packages/@blockprotocol/core/package.json` (or any other) and aligning package metadata.

Reply:
I'm fairly certain I copied the existing `package.json`, so this was already there. I have removed it for consistency though, I agree :)